crashtracker-ffi/cbindgen.toml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+# SPDX-License-Identifier: Apache-2.0
+
+language = "C"
+cpp_compat = true
+tab_width = 2
+header = """// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+"""
+include_guard = "DDOG_CRASHTRACKER_H"
+style = "both"
+pragma_once = true
+
+no_includes = true
+sys_includes = ["stdbool.h", "stddef.h", "stdint.h"]
+includes = ["common.h"]
+
+[export]
+prefix = "ddog_crashtracker_"
+renaming_overrides_prefixing = true
+
+[export.rename]
+"ByteSlice" = "ddog_ByteSlice"
+"CancellationToken" = "ddog_CancellationToken"
+"CharSlice" = "ddog_CharSlice"
+"Endpoint" = "ddog_Endpoint"
+"Error" = "ddog_Error"
+"HttpStatus" = "ddog_HttpStatus"
+"Slice_CChar" = "ddog_Slice_CChar"
+"Slice_I64" = "ddog_Slice_I64"
+"Slice_U8" = "ddog_Slice_U8"
+"Tag" = "ddog_Tag"
+"Timespec" = "ddog_Timespec"
+"Vec_Tag" = "ddog_Vec_Tag"
+"Vec_U8" = "ddog_Vec_U8"
+
+[export.mangle]
+rename_types = "PascalCase"
+
+[enum]
+prefix_with_name = true
+rename_variants = "ScreamingSnakeCase"
+
+[fn]
+must_use = "DDOG_CHECK_RETURN"
+
+[parse]
+parse_deps = true
+include = ["ddcommon", "ddcommon-ffi", "datadog-crashtracker", "ux"]
+
```

Comment:
I went through `crashtracker.h` and here's some of my notes on the APIs:

* `ddog_crashtracker_Configuration` vs `ddog_crashtracker_ReceiverConfig` --> maybe adopt `Config` for both?
* `ddog_crashinfo_new` should probably be `ddog_crashtracker_CrashInfo_new`?
  * Similar note for other crashinfo APIs?
* `ddog_crashinfo_add_file(struct ddog_crashtracker_CrashInfo *crashinfo, ddog_CharSlice name)` -> `name` here is the content to add, not the file name? That seems a bit confusing :eyes: 
* `ddog_crashinfo_set_timestamp` -> should it use a `ddog_Timespec` as argument?

Reply:
- [ ] ddog_crashtracker_Configuration vs ddog_crashtracker_ReceiverConfig --> maybe adopt Config for both
- [ ] ddog_crashinfo_new should probably be ddog_crashtracker_CrashInfo_new?
- [ ] Similar note for other crashinfo APIs?
- [ ] ddog_crashinfo_add_file(struct ddog_crashtracker_CrashInfo *crashinfo, ddog_CharSlice name) -> name here is the content to add, not the file name? That seems a bit confusing 👀
- [ ] ddog_crashinfo_set_timestamp -> should it use a ddog_Timespec as argument?